### PR TITLE
Update to stable EEPROM release

### PR DESCRIPTION
**Pull request by timg236**
Update instructions to refer to the stable release and add some brief notes about USB device enumeration and a health warning about NOOBS

### CODE REVIEW

**Comment by JamesH65** [comment]
LGTM. Going to merge this immediately - if anyone finds any issues later, please use another PR to update.

**Comment by timg236** [comment]
Thanks @JamesH65 this should capture a few FAQs from the forum, improvement PRs always welcome

File: hardware/raspberrypi/bcm2711_bootloader_config.md [hunk 1]
```diff
@@ -295,35 +294,46 @@ sudo reboot
...
 This is especially important with USB SATA adapters which may be supported by the bootloader in mass storage mode but fail if Linux selects [USB Attached SCSI - UAS](https://en.wikipedia.org/wiki/USB_Attached_SCSI) mode. 
 
 See this [forum thread](https://www.raspberrypi.org/forums/viewtopic.php?t=245931) about UAS and how to add [usb-storage.quirks](https://www.kernel.org/doc/html/v5.0/admin-guide/kernel-parameters.html) to workaround this issue.
 
+### Multiple bootable drives
+When searching for a bootable partition the bootloader scans all USB mass storage devices in parallel and will select the first to respond. If the boot partition does not contain a suitable start.elf file the next available device is selected.
+
+As with earlier Raspberry Pi models there is no method for specifying the boot device according to the USB topology because this would slow down boot and adds unecessary and hard to support configuration complexity.
+
+N.B. config.txt [conditional filters])(../configuration/config-txt/conditional.md) can be used to select alternate firmware in complex device configurations.
```
Comment (jamesmyatt) [inline hardware/raspberrypi/bcm2711_bootloader_config.md:320]: extra ")"?

Comment (JamesH65) [reply hardware/raspberrypi/bcm2711_bootloader_config.md:320]: Good spot.

**Pull request merged**